cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/engine/SQLEngineJobTypeMetric.java

```diff
@@ -22,7 +22,12 @@
 public enum SQLEngineJobTypeMetric {
   JOIN("join"),
   TRANSFORM("transform"),
-  WRITE("join");
+  PUSH("push"),
+  PULL("pull"),
+  SPARK_PUSH("spark_push"),
+  SPARK_PULL("spark_pull"),
+  READ("read"),
```

Comment:
wherr are we using this id it is a new addition?

Reply:
Direct reads in the future